be/src/vec/exec/join/vhash_join_node.cpp

```diff
@@ -509,11 +517,6 @@ struct ProcessHashTableProbe {
                 }
                 output_block->get_by_position(result_column_id).column =
                         std::move(new_filter_column);
-            } else if constexpr (JoinOpType::value == TJoinOp::RIGHT_OUTER_JOIN) {
```

Comment:
why delete this code?

Reply:
> why delete this code?
`RIGHT_OUTER_JOIN` is toe to toe with `RIGHT_SEMI_JOIN/RIGHT_ANTI_JOIN` now.